capstone/scripts/convert_s3.py

```diff
@@ -89,123 +89,135 @@ def export_cases_to_s3(redacted: bool, reporter_id: str) -> None:
     put_reporter_metadata(bucket, reporter, reporter_prefix)
 
     # get volumes in reporter
-    volumes = VolumeMetadata.objects.select_related().filter(reporter=reporter_id)
-    # export volume metadata/cases
-    export_cases_by_volume(volumes, bucket, redacted)
+    volumes = list(reporter.volumes.select_related())
```

Comment:
I think this can just be `for volume in reporter.volumes` -- you're not touching each volume more than once, so `list` isn't necessary, and I don't see where `select_related` is being used.

Reply:
Oh but we also need some filtering on which volumes and reporters are in scope, right?

Reply:
There aren't out of scope reporters, afaict. There are those with no volume_count, but then we'd still want the reporter metadata file I believe.

Reply:
My guess is a reporter without volumes doesn't have any useful metadata either and we just exclude it.